src/lnp/application/message.rs

```diff
@@ -481,7 +481,7 @@ pub struct ClosingSigned {
     Clone, PartialEq, Eq, Debug, Display, LightningEncode, LightningDecode,
 )]
 #[lnpbp_crate(crate)]
-#[display(Debug)]
+#[display("update_add_htlc({channel_id}, {htlc_id}, {asset_id:#?})")]
```

Comment:
amount and payment hash are very important

Reply:
Done